workers.go

```diff
@@ -89,6 +89,8 @@ func (b WorkerBindingType) String() string {
 }
 
 const (
+	// WorkerDurableObjectBindingType is the type for Durable Object bindings.
+	WorkerDurableObjectBindingType WorkerBindingType = "durable_object_namespace"
```

Comment:
I put this at the top to try for sorted order. But I noticed that the Service Workers was added to the end of the list and Wasm has been in the middle for a while. lmk if I should follow suit or if this works.